Library/Formula/qt5.rb

```diff
@@ -43,20 +51,15 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :xcode => :build
 
-  # There needs to be an OpenSSL dep here ideally, but qt keeps ignoring it.
-  # Keep nagging upstream for a fix to this problem, and revision when possible.
-  # https://github.com/Homebrew/homebrew/pull/34929
-  # https://bugreports.qt.io/browse/QTBUG-42161
-  # https://bugreports.qt.io/browse/QTBUG-43456
-
   depends_on OracleHomeVarRequirement if build.with? "oci"
 
   def install
     ENV.universal_binary if build.universal?
 
     args = ["-prefix", prefix,
-            "-system-zlib",
+            "-system-zlib", "-securetransport",
             "-qt-libpng", "-qt-libjpeg",
+            "-no-rpath", "-no-openssl",
```

Comment:
I'm not completely sure what's going on here. `rpath` [apparently became](https://github.com/Homebrew/homebrew/pull/41252#issuecomment-118557153) the default for Qt5 in this release, and the bot errors imply (Although perhaps not correctly) that the `rpath` setting wasn't linking as expected for some reason.

I did a full build locally running with the same `--build-bottle` variables as the test-bot and both the installation and bottle turned out fine, which makes me wonder if I'm getting lucky or if something about the test-bot is borking the installation in some way. We disable rpath usage in a few formulae, but it isn't widespread. There are build logs from my local install [here](https://gist.github.com/anonymous/d2047f4c8db4e58adc15) if that's any use at all.